Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/CoordinatedGraphicsScene.cpp

```diff
@@ -30,12 +30,14 @@
 #include <WebCore/NicosiaScene.h>
 #include <WebCore/TextureMapperLayer.h>
 #include <wtf/Atomics.h>
+#include <wtf/SystemTracing.h>
 
 namespace WebKit {
 using namespace WebCore;
 
-CoordinatedGraphicsScene::CoordinatedGraphicsScene(CoordinatedGraphicsSceneClient* client, Damage::ShouldPropagate propagateDamage)
+CoordinatedGraphicsScene::CoordinatedGraphicsScene(CoordinatedGraphicsSceneClient* client, const WebCore::Settings& settings, Damage::ShouldPropagate propagateDamage)
     : m_client(client)
+    , m_settings(settings)
```

Comment:
I don't think we should keep settings as a member. Let's just pass the settings we need to the methods using them, or to the constructor and keep the setting value, not the settings object, liked currently do for m_propagateDamage